warehouse-management-service/internal/config/config_test.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestFromFile(t *testing.T) {
+	config, err := FromFile("./testdata/config.json")
+	wantConfig := &Config{
+		LogLevel: "debug",
+		Postgres: PostgresConfig{
+			Host:     "localhost",
+			Port:     "5432",
+			Username: "user",
+			Password: "xxxxxxxx",
+			DBName:   "db",
+			SSLMode:  "disable",
+		},
+		DBMigration: DBMigration{SourcePath: "file://warehouse-management-service/db/migrations"},
+	}
+
+	if err != nil || !reflect.DeepEqual(config, wantConfig) {
+		t.Errorf("want config: %v, err: %v, got config: %v, err: %v", wantConfig, nil, config, err)
+	}
```

Comment:
[tiny ergonomic nit]
Since these are two different assertions, separating them like this would make it easier to parse what went wrong if something fails.

```suggestion
	if err != nil {
		t.Errorf("want err: %v, got err: %v", nil, err)
	}
	if !reflect.DeepEqual(config, wantConfig) {
		t.Errorf("want config: %v, got config: %v", wantConfig, config)
	}
```